### ISSUE TITLE
[Android] Opt-out cookie consent notice

### ISSUE THREAD

**Issue by timchilds** [brave/brave-browser#31767]
## Description
Implement the cookie consent notice as opt-out. Same display logic as the old notice before it was disabled.

![image](https://github.com/brave/brave-browser/assets/28269287/5d3f791f-4419-4ac5-b255-999aa0d1263c)
_Final copy TBD_

- Got it = keep cookie consent filter enabled (it's enabled by default).
- Stop blocking = disable cookie consent filter.

### Figma
https://www.figma.com/file/cL22PpqneTYxJePORxUBKz/Cookie-consent-blocker?type=design&node-id=1242%3A44596&mode=design&t=FSGniL2oAsfIVxUT-1

cc @deeppandya @anthonypkeane

**Comment by deeppandya** [comment]
below is the final copy for the issue ,


![Image](https://github.com/brave/brave-browser/assets/5146988/4170e30f-32fd-468a-bd4f-9a031fe3d0d4)

**Comment by DanielProg39** [comment]
Well, I think that there should be an option in Brave Shield to either block cookie consent notices or not so that we are able to turn it on/off at any time

**Comment by kjozwiak** [comment]
Labelling this as `QA/Blocked` as we'll be removing the cookie consent modal as per previous discussions. With the above issue, we can basically confirm that the modal should never appear. Believe @deeppandya is currently working on a PR and will merge into `master` and uplift into `1.59.x`. Once the fix lands, the feature should behave/look as per the following:

* we should never display any type of modal re: the cookie consent feature
* it should be enabled by default on startup (feature will be opt-out and not reliant on Griffin anymore)
* we should always display the cookie consent settings via `Settings` (also fix https://github.com/brave/brave-browser/issues/32949)

**Comment by kjozwiak** [comment]
Removing `QA/Blocked` and labelling as `QA/No` as we'll use https://github.com/brave/brave-browser/issues/33151 to verify that the cookie consent modal has been removed which will be uplifted into `1.59.x` via
